tst/PortingAssistantExtensionIntegTests/TestProjects/MvcMusicStore.json

```diff
@@ -140,15 +140,15 @@
       {
         "fileName": "StoreManagerController.cs",
         "compatCode": "PA0002",
-        "message": "Replace System.Web.Mvc namespace with Microsoft.AspNetCore.Mvc.",
+        "message": "Replace System.Data.Entity namespace with Microsoft.EntityFrameworkCore. Also, package Microsoft.EntityFrameworkCore will be added.",
```

Comment:
Is this change due to a recent rule file update in [porting-assistant-dotnet-datastore](https://github.com/aws/porting-assistant-dotnet-datastore)?

Reply:
Most probably